lib/tether/tether.go

```diff
@@ -252,6 +258,46 @@ func (t *tether) setMounts() error {
 			return fmt.Errorf("unsupported volume mount type for %s: %s", k, v.Source.Scheme)
 		}
 	}
+	return t.populateVolumes()
+}
+
+func (t *tether) populateVolumes() error {
+	defer trace.End(trace.Begin(fmt.Sprintf("populateVolumes")))
+	// skip if no mounts present
+	if len(t.config.Mounts) == 0 {
+		return nil
+	}
+	// skip if this was done before
+	if _, err := os.Stat(mountsCopied); err == nil {
```

Comment:
Echo'ing @hickeng here.  For NFS volumes where 2 containers mount the same volume, the 2nd container will attempt to copy the contents of the mount point *again*.  I thought we discussed this and were simply going to skip copying anything into the volume if the volume was non-empty.